freqtrade/plugins/pairlist/AgeFilter.py

```diff
@@ -63,7 +68,7 @@ def filter_pairlist(self, pairlist: List[str], tickers: Dict) -> List[str]:
 
         since_ms = int(arrow.utcnow()
                        .floor('day')
-                       .shift(days=-self._min_days_listed - 1)
+                       .shift(days=-self._max_days_listed - 1)
```

Comment:
with this change, you'll "attempt" to download history since 7300 days by default.
For each pair - that's 7-14 requests (average answers are between 500 and 1000 candles, depending on the exchange).

This will make this filter terribly slow - especially considering that it'll run on every startup for every pair.
Starting this on kraken would probably not work (well, it would work technically, but not within my patience realm) - as kraken can only take ~1 request every 3 seconds).

Reply:
should we make it dynamic? if we put `max_days_listed` on the config, then use `max_days_listed` else use `min_days_listed`
what do you think?

Reply:
it's not so much the amount - but the way it works that's going to be problematic if it starts doing multiple exchange calls

with `min_days_listed` - I'd expect people to usually not go further back than ~500 days (which is 1 api call).
with `max_days_listed` this might be different.

If you notice above (in `__init__`) - there's an exception/error when `min_days_listed` is > one exchange call.
Realistically, we should probably have the same "blocker" for `max_days` to avoid having AgeFilter take forever when it's starting

Reply:
is it enough when we add same "blocker" like `min_days_listed` to `max_days_listed` ?
if the issue is on the api call then what should we do ? I was searching binance api doc to get "listed date", but I got nothing. I am not sure it will be in ccxt

Reply:
no, the only way (i found) to get this is to download candles from "right before" the desired date and check if there's data.

for binance - you can request data from 4 years ago - if the pair was listed in 2020, you'll simply get that data - so an alternative would be to download 1 call from the exchange and look at the dates instead of dataframe length.

The problem with the way AgeFilter is working currently is that it'd download all data since 2020 - which would not be necessary for AgeFilter.
However - the exchange class currently has no possibility to fetch only 1 call (or define an end-date) - so the call we use will always download from "since" to today.

Reply:
yes I agree with you, for now limit to 1 call and we can to keep it as "further enhancement"